AdblockPlusSafari/AdblockPlus.h

```diff
@@ -17,22 +17,28 @@
 
 #import <Foundation/Foundation.h>
 
+extern NSString *_Nonnull AdblockPlusActivated;
+
 @interface AdblockPlus : NSObject
 
+@property (nonatomic, strong, readonly) NSUserDefaults *__nonnull adblockPlusDetails;
+
+- (NSString *__nonnull)group;
+
+- (NSString *__nonnull)contentBlockerIdentifier;
+
+- (NSString *__nonnull)backgroundSessionConfigurationIdentifier;
+
 @property (nonatomic) BOOL enabled;
 
 @property (nonatomic) BOOL acceptableAdsEnabled;
 
 @property (nonatomic) BOOL activated;
 
-@property (nonatomic) BOOL reloading;
-
-- (void)setEnabled:(BOOL)enabled reload:(BOOL)reload;
-
-- (void)setAcceptableAdsEnabled:(BOOL)enabled reload:(BOOL)reload;
+@property (nonatomic) NSInteger installedVersion;
```

Comment:
Maybe I'm missing something, but I get the impression we could reduce complexity here and achieve the same effect by replacing the `installedVersion` and `downloadedVersion` counters with a single boolean `dirty` flag (actually not sure about the best name, `refreshed` or `applied` also come to mind).